webpack/assets/javascripts/react_app/components/common/ConfirmDialog/ConfirmDialog.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'patternfly-react';
+import { translate as __ } from '../../../common/I18n';
+import Dialog from '../Dialog';
+
+const ConfirmDialog = props => {
```

Comment:
@boaz0 does that overlap with your delete dialog in #6301?

Reply:
It seems like it doesn't really overlap. However, I am not sure about reusing this.
@MariaAga why did you choose to use `Modal` over [`MessageDialog`](http://patternfly-react.surge.sh/patternfly-3/index.html?knob-Icon=info&knob-With%20Title=true&knob-With%20Primary%20Content=true&knob-With%20Secondary%20Button=true&knob-With%20Icon=true&knob-With%20Secondary%20Content=true&selectedKind=patternfly-react%2FCommunication%2FMessage%20Dialog&selectedStory=Message%20Dialog%20with%20Toggleable%20Options&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs)?

Reply:
@boaz0 if you mean using `Modal` in the `Dialog` component then I didn't choose `Modal`, it was in the code, and in this PR I only moved components from Katello to Foreman without changing them.

Reply:
@MariaAga I see. That's what I thought.
Anyway, to me it looks like that PF3 has implemented this component: `MessageDialog` and we should use it instead of this.

If we can do this in here then great. Otherwise, lets see how we can align with PF3 as soon as possible.